src/sources/prometheus/scrape.rs

```diff
@@ -156,6 +171,20 @@ fn prometheus(
                 auth.apply(&mut request);
             }
 
+            let instance_info = instance_key.as_ref().map(|key| {
+                let instance = format!("{}:{}", url.host().unwrap_or_default(), url.port_u16().unwrap_or_else(|| {
+                    let scheme = url.scheme();
+                    if url.scheme() == Some(&http::uri::Scheme::HTTP) {
+                        80
+                    } else if scheme == Some(&http::uri::Scheme::HTTPS) {
+                        443
+                    } else {
+                        0
+                    }
+                }));
+                InstanceInfo{ key: key.to_string(), instance }
```

Comment:
Did this miss a `rustfmt`?

Reply:
Ah, yes, my local environment was acting weird yesterday and not actually doing the autoformatting. I thought I fixed it, but obviously not 😅

Reply:
Turns out this seems to be a bug in rustfmt. I was able to workaround it via: 575c35e